[2024-06-24T16:04:29.218+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T16:04:29.292+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2021-08-10T00:00:00+00:00 [queued]>
[2024-06-24T16:04:29.329+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2021-08-10T00:00:00+00:00 [queued]>
[2024-06-24T16:04:29.332+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-24T16:04:29.378+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task1> on 2021-08-10 00:00:00+00:00
[2024-06-24T16:04:29.405+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'scheduled__2021-08-10T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmptq_iha4l']
[2024-06-24T16:04:29.404+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=282) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T16:04:29.408+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask task1
[2024-06-24T16:04:29.408+0000] {standard_task_runner.py:63} INFO - Started process 289 to run task
[2024-06-24T16:04:29.621+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag.task1 scheduled__2021-08-10T00:00:00+00:00 [running]> on host 7608b9084855
[2024-06-24T16:05:03.125+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 14 for task task1 ((MySQLdb.OperationalError) (2005, "Unknown server host 'mysql' (-3)")
(Background on this error at: https://sqlalche.me/e/14/e3q8); 289)
[2024-06-24T16:05:05.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-24T16:05:05.547+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T16:05:05.549+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
